README.md

```diff
@@ -211,6 +220,15 @@ sortable('.sortable', {
 });
 ```
 
+### containerClass
```

Comment:
I like the idea. What we need to do now is to improve the naming to make it more obvious when this class is added.
What do you think about `dropTargetContainerClass`? I know it is a mouthful. I am open for better ideas, I just want the class to reflect its function more.